src/Trixi.jl

```diff
@@ -105,6 +105,7 @@ include("auxiliary/p4est.jl")
 include("equations/equations.jl")
 include("meshes/meshes.jl")
 include("solvers/solvers.jl")
+include("equations/parabolic_equations.jl") # these depend on parabolic solver types
```

Comment:
See comment below on where to put stuff depending on the equations and the solver.

Reply:
👍